docs/using/codeless.rst

```diff
@@ -0,0 +1,204 @@
+.. include:: ../links.inc
+
+.. _codeless:
+
+Code-less configuration
+=======================
+
+On of the most important features of junifer is its capacity to run without writing a single line of code. This is
+achieved by using a configuration file that is written in YAML_. In this file, we configure the different steps of
+:ref:`pipeline`.
+
+As a reminder, this is how the pipeline looks like:
+
+.. image:: ../images/pipeline/pipeline.001.png
+
+Thus, the configuration file must configure each of the sections of the pipeline, as well as some general parameters.
+
+As an example, we will generate the configuration file for a pipeline that will extract the mean ``VBM_GM`` values
+using two different parcellations and one set of coordinates, from the *Oasis VBM Testing dataset* included in 
+junifer.
+
+
+General Parameters
+------------------
+
+The general parameters are the ones that are not specific to any of the sections of the pipeline, but configure
+junifer as a whole. These parameters are:
+
+* ``with``: A section used to specify modules and junifer extensions to use.
+* ``workdir``: The working directory where junifer will store temporary files.
+
+Since the example uses a specific datagrabber for testing, we need to add ``junifer.testing.registry`` to the
+``with`` section. This will allow junifer to find the datagrabber. We will set the ``workdir`` to ``/tmp``.
+
+.. code-block:: yaml
+
+  with: junifer.testing.registry
+  workdir: /tmp
+
+Step-by-step configuration
+--------------------------
+
+In order to configure the pipeline, we need to configure each step:
+
+* ``datagrabber``
+* ``datareader``
+* ``preproces``
+* ``markers``
+* ``storage``
+
+.. important:: The datareader step configuration is optional, as junifer only provides one datareader. Nevertheless,
+    it is possible to extend junifer with custom datareaders, and thus, it is also possible to configure this step.
+
+
+Data Grabber
```

Comment:
`DataGrabber`

Reply:
will keep it as concepts and not class names